### PR TITLE
chore: add basic integration test

### PR DESCRIPTION
**Pull request by AlekSi**
It will be improved later.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -106,3 +108,26 @@ run: install ## Run the controller locally. This is for testing purposes only.
...
+	curl -Lo talosctl https://github.com/talos-systems/talos/releases/download/$(TALOS_VERSION)/talosctl-$(shell uname -s | tr "[:upper:]" "[:lower:]")-amd64
+	chmod +x ./talosctl
+
+env-up: talosctl
+	./talosctl cluster create \
+		--name=cabpt-env \
+		--kubernetes-version=$(K8S_VERSION) \
+		--mtu=1450 \
+		--memory=2048 \
+		--cpus=2.0 \
```
Comment (AlekSi) [inline Makefile:127]: Both `memory` and `cpus` use default values, do we have to specify them explicitly?

Comment (smira) [reply Makefile:127]: That was me fighting with the build. We can keep it or omit it, I don't thinkt it matters a lot

**Review by smira — APPROVED** [review]
Tests seem to be a bit flaky, but it's better than not having any. We can improve them as we add more tests

**Pull request merged**